support/mock/builder.go

```diff
@@ -80,3 +81,13 @@ func (b *RuntimeBuilder) WithBalance(balance, received abi.TokenAmount) *Runtime
 	b.rt.valueReceived = received
 	return b
 }
+
+func (b *RuntimeBuilder) WithReceiverType(code cid.Cid) *RuntimeBuilder {
```

Comment:
This doesn't make sense to me, the receiving actor should never be inspecting its own type. But if you make a `WithActorType(addr, cid)` it might make more sense to me. And maybe this will drop out after the constructor is fixed.